### ISSUE TITLE
Pixel size setting can misalign canvas

### ISSUE THREAD

**Issue by TodePond** [munshkr/flok#305]
I can repro like this. Probably device dependent. MacOS Chrome. My screen as 2 DPI.

1. Make sure your pixel size setting is set to 1.
2. Put a rectangle on the screen, eg:

```js
shape().out()
```

3. Make sure it's visible and in the middle of the screen.
4. Change the pixel size setting to 2.

Expected: Rectangle is still within the middle of the screen.
Actual: Rectangle is not visible.

Interestingly, this works fine:
```js
shape().scrollX(0).out()
```

Will investigate this further!

**Comment by sqaxomonophonen** [comment]
The canvas sizing happens here:

https://github.com/sqaxomonophonen/flok/blob/4535ae759b38f422f52e93481c1261213b5af81c/packages/web/src/components/hydra-canvas.tsx#L21-L23

Before my patch it was:
```
      width={window.innerWidth}
      height={window.innerHeight}
```

I suspect the misalignment may also happen when resizing the window?

**Comment by TodePond** [comment]
> I suspect the misalignment may also happen when resizing the window?

Yes I imagine so! I was trying out some fixes but got distracted by #307 instead :)
